www/webapp/package.json

```diff
@@ -15,6 +15,7 @@
     "axios": "~0.27.2",
     "core-js": "~3.23.4",
     "javascript-time-ago": "^2.5.6",
+    "qrcode.vue": "~1.7.0",
```

Comment:
As far as I can tell, the suggested library has much worse maintenance (last release 5 years ago, https://www.npmjs.com/package/vue-qrcode-component). In fact, it is merely a VueJS component wrapper around its run-time dependency on @keeex/qrcodejs-kx.

I share the concern and spent considerable time finding the right library. The one I chose uses qr.js which it depends on during build time, where it is inlined. I thought that's better.

Everything is under MIT license, and nothing is expected to change (the background QR code generation dependencies of the two libraries are 5 and 9 years old, respectively). Perhaps it would be best to just keep the whole thing inlined in our repo. Should not be too difficult. What do you think?

Reply:
Let's only inline it if we can't pin it using a cryptographic hash?

Reply:
I'm not sure how to do that. Even if we can pin the Vue component module, if *their* dependency is not hash-pinned, we may get a different version of the underlying QR library with the next build. - Any ideas?

Reply:
You are right - it does not appear in `package-lock.json`. I'm okay with inlining it.

Reply:
done! d6af28657e3e7b210f07d63977cf4191e52ae089